sdk/servicebus/azure-servicebus/azure/servicebus/_pyamqp/aio/_transport_async.py

```diff
@@ -540,7 +540,11 @@ async def _read(self, toread, buffer=None, **kwargs):  # pylint: disable=unused-
                     self._read_buffer = BytesIO(data[toread:])
                     toread = 0
             return view
-        except:
+        except TypeError as te:
+            # aiohttp websocket raises TypeError when a websocket disconnects, as it ends up
+            # reading None over the wire and cant convert to bytes.
+            raise ConnectionError("Websocket disconnected: %r" % te) from None
+        except Exception:
```

Comment:
just double checking but the TypeError isn't useful at all?

Reply:
No,  its not useful, it just says `TypeError: Received message 257:None is not bytes`, as we are doing `receive_bytes` and it fails the type check. Its safe to assume if None is read in that the connection has dropped